lua/nvim-tree/actions/node/open-file.lua

```diff
@@ -164,13 +164,13 @@ local function on_preview(buf_loaded)
   view.focus()
 end
 
-local function get_target_winid(mode, win_ids)
+local function get_target_winid(mode)
   local target_winid
   if not M.window_picker.enable or mode == "edit_no_picker" then
     target_winid = lib.target_winid
 
     -- first available window
-    if not vim.tbl_contains(win_ids, target_winid) then
+    if not vim.tbl_contains(vim.api.nvim_tabpage_list_wins(0), target_winid) then
```

Comment:
:+1: 0 is current tabpage

Reply:
do you want me to explicitly get_current_tabpage? i see that's how its done in some other places in the code but I don't really know why when 0 works

Reply:
That's fine. It was probably a copy/paste from a location that was also using tabpage.